src/dotnet/commands/dotnet-test/LocalizableStrings.cs

```diff
@@ -55,5 +55,7 @@ internal class LocalizableStrings
 
         public const string RunSettingsArgsHelpText = @"Any extra commandline runsettings arguments that should be passed to vstest. See 'dotnet vstest --help' for available options.
                                         Example: -- RunConfiguration.ResultsDirectory=""C:\users\user\desktop\Results Directory"" MSTest.DeploymentEnabled=false";
+        public const string CmdResultsDirectoryDescription = @"Test results directory will be created in specified path if not exists.
+                                        Example: --results-directory <PATH_TO_RESULTS_DIRECTORY>";
```

Comment:
I'd suggest a slight change to the message text: "The test results directory will be created in the specified path if it does not exist."